app/src/main/java/com/sunasterisk/a14day_challenge/screen_main/SplashActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.sunasterisk.a14day_challenge.screen_main
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.sunasterisk.a14day_challenge.screen_main.login.LoginActivity
+
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val intent = Intent(this, LoginActivity::class.java)
+        startActivity(intent)
```

Comment:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md
conventions 2.11 
Em tạo 1 hàm getIntent trong LoginActivity, hàm này trả về Intent tới màn hình LoginActivity 
ở đây em chỉ gọi 
```suggestion
        startActivity(LoginActivity.getIntent(this))
```

Reply:
Dạ vâng khi nào design LoginActivity em sẽ sửa ạ

Reply:
sửa luôn nhé